cmd/csi/init/cmd.go

```diff
@@ -82,6 +81,6 @@ func run() func(*cobra.Command, []string) error {
 	}
 }
 
-func createCSIDataPath(fs afero.Fs) error {
-	return errors.WithStack(fs.MkdirAll(dtcsi.DataPath, os.ModePerm))
+func createCSIDataPath() error {
```

Comment:
i see this helper implemented a few times. would it make sense to move it to a shared location?

Reply:
done